kie-api/src/main/java/org/kie/api/runtime/RequestContext.java

```diff
@@ -36,4 +36,12 @@ static RequestContext create() {
             throw new RuntimeException("Unable to instance RequestContext", e);
         }
     }
+    
+    static RequestContext create(ClassLoader classLoader) {
+        try {
+            return (RequestContext) Class.forName( "org.drools.core.command.RequestContextImpl", true, classLoader ).newInstance();
+        } catch (Exception e) {
+            throw new RuntimeException("Unable to instance RequestContext", e);
+        }
+    }
```

Comment:
Could we remove the duplication and call the other method? E.g. `create()` would call `create(ClassLoader)` with the default classloader.